docs/topics/mta-about-console-guide.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+
+:_content-type: CONCEPT
+[id="mta-about-console-guide_{context}"]
+= About the User Interface Guide
+
+This guide is for engineers, consultants, and others who want to use the {ProductName} ({ProductShortName}) {WebName} to accelerate large-scale application modernization efforts across hybrid cloud environments on Red Hat OpenShift. This solution provides insight throughout the adoption process, at both the portfolio and application levels: inventory, assess, analyze, and manage applications for faster migration to OpenShift via the user interface.
```

Comment:
I would also add  "architects" to the list.

Reply:
Done.